### ISSUE TITLE
Improve appearance of SPDX License List landing page

### ISSUE THREAD

**Issue by jlovejoy** [spdx/license-list-XML#1543]
could we improve the top of the pages at https://spdx.org/licenses/ and https://spdx.org/licenses/exceptions-index.html to have a more graphical appearance instead of a block of bullets - maybe a tile for each bullet or something like that.

**Comment by jlovejoy** [comment]
need to check on status of potential website update/move to Github before we bother making changes here/in current paradigm. Hold off on this

**Comment by xsuchy** [comment]
I disagree. I like this page. It is clean and simple. Good for technical description, which this page should provide.

**Comment by jlovejoy** [comment]
given the age of this issue and that we haven't addressed it yet, but it's something we could easily revisit in the future. I'm closing this!
